docs/cloud-soar/global-functions-menu.md

```diff
@@ -237,14 +237,13 @@ before they are converted to incidents. The Triage module can be
 completely customized for use cases from financial fraud to network IDS
 alerts.
 
-<!-- Add this once the Reports feature released to production. 
 ## Report
 
-You can create reports on incidents to share with others. You can also [create widgets](/docs/cloud-soar/main-menu#create-widgets-for-dashboards-or-reports) to use in the report that display text, graphs, and charts containing details about incidents and other aspects of Cloud SOAR.
+With the **Report** option, you can create incident reports to share with others as well as [widgets](/docs/cloud-soar/main-menu/#create-widgets) to use in the report that display text, graphs, and charts containing details about incidents and other aspects of Cloud SOAR.
 
 1. Click the gear icon in the upper-right corner of the UI, then select **Report**. <br/><img src={useBaseUrl('img/cloud-soar/delivery-2-access-reports.png')} alt="Access reports" width="150"/><br/>The Report UI appears. <br/><img src={useBaseUrl('img/cloud-soar/delivery-2-report-ui.png')} alt="Reports user interface" width="600"/> 
 1. Click the **+** icon in the upper left corner. 
-1. On the right side, select widgets to add to the report from **My Widgets** or **Public**. These are the same widgets that are available to use in [dashboards](/docs/cloud-soar/main-menu#create-a-dashboard). Widgets can be graphs, charts, tables, or any kind of visual element that contains information. Click **New** to [create a new widget](/docs/cloud-soar/main-menu#create-widgets-for-dashboards-or-reports). Click **Show List** to see all available widgets.  
+1. On the right side, select widgets to add to the report from **My Widgets** or **Public**. These are the same widgets that are available to use in [dashboards](/docs/cloud-soar/main-menu/#create-a-dashboard). Widgets can be graphs, charts, tables, or any kind of visual element that contains information. Click **New** to [create a new widget](/docs/cloud-soar/main-menu/#create-widgets). Click **Show List** to see all available widgets.  
```

Comment:
@kimsauce, when I click the "create a new widget" or "dashboards" links, sometimes it doesn't jump completely to the target sections. I've seen this linking problem happen sporadically. Is there anything we can do to fix it?

Reply:
@kimsauce, thank you for letting me know about this possible fix:
https://docusaurus.io/docs/api/themes/configuration#auto-hide-sticky-navbar

Reply:
Per our convo this morning, this is a [known docusaurus issue](https://github.com/facebook/docusaurus/issues/2907). Just committed [this](https://github.com/SumoLogic/sumologic-documentation/pull/3701/commits/02c047a1517e471bb9271b3c77185a181ce441ea) (fingers crossed) fix!

Reply:
@kimsauce, I tried it out in my local build, and the `hideOnScroll: true,` worked like a charm! However, it might be a while before I can merge this PR. Could we put that fix into its own PR? That way, if it causes any issues, we can revert it easily.

Reply:
done! https://github.com/SumoLogic/sumologic-documentation/pull/3715

Reply:
Unfortunately, @kimsauce, PR #3715 didn't fix the linking problem. For an example of an article containing links that show the problem, see this [release note](https://help.sumologic.com/release-notes-csoar/2024/02/06/application-update/).